src/components/CompletingParty/CompletingParty.stories.ts

```diff
@@ -0,0 +1,73 @@
+import { CompletingParty } from './index'
+import Vuetify from 'vuetify'
+import { required, maxLength } from 'vuelidate/lib/validators'
+
+export default {
+  title: 'component/CompletingParty',
+  component: CompletingParty,
+  argTypes: {
+  }
+}
+
+const PersonAddressSchema = {
+  streetAddress: {
+    required,
+    maxLength: maxLength(50)
+  },
+  streetAddressAdditional: {
+    maxLength: maxLength(50)
+  },
+  addressCity: {
+    required,
+    maxLength: maxLength(40)
+  },
+  addressCountry: {
+    required
+  },
+  addressRegion: {
+    maxLength: maxLength(2)
+  },
+  postalCode: {
+    required,
+    maxLength: maxLength(15)
+  },
+  deliveryInstructions: {
+    maxLength: maxLength(80)
+  }
+}
+
+const Template = (args, { argTypes }) => ({
+  vuetify: new Vuetify({ iconfont: 'mdi' }),
+  props: Object.keys(argTypes),
+  components: { CompletingParty },
+  template: '<v-card flat ><completingParty v-bind="$props" /></v-card>'
```

Comment:
I know Vue is "flexible" with component naming but I think this should be `completing-party` or `CompletingParty`.

Reply:
oops, for sure, will fix to `CompletingParty`